clients/drcachesim/analyzer_multi.cpp

```diff
@@ -121,7 +121,8 @@ analyzer_multi_t::analyzer_multi_t()
                 dir.modfile_bytes_, dir.in_files_, dir.out_files_, dir.out_archives_,
                 dir.encoding_file_, dir.serial_schedule_file_, dir.cpu_schedule_file_,
                 nullptr, op_verbose.get_value(), op_jobs.get_value(),
-                op_alt_module_dir.get_value(), op_chunk_instr_count.get_value());
+                op_alt_module_dir.get_value(), op_chunk_instr_count.get_value(),
+                dir.in_kfiles_map_, dir.kcoredir_, dir.kallsymsdir_);
```

Comment:
We should have done this earlier: this arg list is getting unwieldy being this long.  We should have an options struct instead.

Reply:
Probably such a change should be done separately and does involve changes to external subclasses.  Add a TODO and file an issue?

Reply:
Create a new feature request [issue#6145](https://github.com/DynamoRIO/dynamorio/issues/6145).